etc/LinksOfAllHelpSections.g

```diff
@@ -10,8 +10,6 @@
 WriteAllLinksOfAllHelpSections := function()
   local all, res, rootpath, entries, entry, path, book, a;
 
-  LoadAllPackages();
```

Comment:
Are you sure that all package manuals will be loaded, even if packages are not loaded? Seems the case now (no way to delay that in https://github.com/gap-packages/example/blob/master/PackageInfo.g) - I guess that was not so some time before, though...

Reply:
I don't know. At the moment, it seems like all of the expected manuals appear.

The reason I did this is because I couldn't load all of the packages. I think this is because:

In order to actually load all of the packages, you have to compile all of the packages, and in order to compile some of the packages, you need particular software tools installed that I don't have.

From my opinion, it is undesirable to require the computer that runs the 'update the website script' (whether it's someone's local machine, or on GitHub Actions eventually) to have to have such software installed.

Reply:
Actually, I can leave the `LoadAllPackages();` line in, it will be harmless as long as I haven't tried compiling all packages with the `bin/BuildPackages.sh` script. However, this means that many packages won't load so it seems quite pointless.

However, once I have attempted to compile all packages (and failed to compile some of them, such as `xgap`) then `LoadAllPackages();` enters a break loop:
```
Error, found no Singular executable in PATH while searching the following list:
[ "Singular" ]
 at /private/tmp/wilf_experiment/gap-4.11.0/pkg/IO_ForHomalg-2019.09.02/gap/IO\
_ForHomalg.gi:309 called from
CallFuncList( LaunchCAS_IO_ForHomalg, Concatenation( [ HOMALG_IO_CAS ],
   arg{[ 2 .. nargs ]} )
 ) at /private/tmp/wilf_experiment/gap-4.11.0/pkg/HomalgToCAS-2019.12.08/gap/I\
O.gi:126 called from
LaunchCAS( HOMALG_IO_CAS
 ) at /private/tmp/wilf_experiment/gap-4.11.0/pkg/HomalgToCAS-2019.12.08/gap/H\
omalgExternalRing.gi:448 called from
CallFuncList( CreateHomalgExternalRing, ar
 ) at /private/tmp/wilf_experiment/gap-4.11.0/pkg/RingsForHomalg-2019.12.08/ga\
p/Singular.gi:1137 called from
CallFuncList( RingForHomalgInSingular, R
 ) at /private/tmp/wilf_experiment/gap-4.11.0/pkg/RingsForHomalg-2019.12.08/ga\
p/Singular.gi:1358 called from
<function "HomalgFieldOfRationalsInSingular">( <arguments> )
 called from read-eval loop at /private/tmp/wilf_experiment/gap-4.11.0/pkg/NumericalSgps-1.2.1/gap/affine-\
extra-gm.gi:9
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
brk>
```
So from my point of view, since things seem to work correctly by omitting the `LoadAllPackages();`, it is best to do so for now.

Reply:
@wilfwilson agree

Reply:
@wilfwilson checked this - indeed, that must be a relic from old times when packages had books which were not loaded. So safe to remove LoadAllPackages call.
```
gap> HELP_KNOWN_BOOKS[1];
[ "tutorial", "reference", "4ti2interface not loaded", "ace not loaded", "aclib", "agt not loaded", "alnuth", "anupq not loaded", 
  "atlasrep", "autodoc", "automata not loaded", "automgrp not loaded", "autpgrp", "browse not loaded", "cap not loaded", 
  "caratinterface not loaded", "cddinterface not loaded", "circle not loaded", "cohomolo not loaded", "congruence not loaded", 
...
```

Reply:
Thanks for looking into this.